portfolio/src/main/webapp/index.html

```diff
@@ -50,6 +51,21 @@ <h1 class="section-title" id="fun-quiz">Superfun quiz</h1>
         <section id="quiz">
             <button onclick="initGame()" class="quiz-button">Start</button>
         </section>
+        <h1 class="section-title" id="comments">Comments</h1>
+        <section>
+            <label for="max-comments">Max number of comments on page:</label>
+            <input type="text" name="max-comments" id="max-comments">
+            <div class="comments-container"></div>
+            <form action="/comments" method="POST" id="comment-add-form">
+                <p class="comment-input-title">Add your comment:</p>
+                <label for="comment-owner">Name: </label>
+                <input type="text" class="comment-input-owner" name="comment-owner">
+                <label for="comment-text">Comment: </label>
+                <textarea class="comment-input-text" name="comment-text"></textarea>
```

Comment:
Could you resize this to match the text around it?
It seems disproportionate now.